app/app.go

```diff
@@ -374,6 +388,7 @@ func New(
 		ibc.NewAppModule(app.IBCKeeper),
 		params.NewAppModule(app.ParamsKeeper),
 		transferModule,
+		nftModule,
```

Comment:
The only potential change I saw that we could make would be to change this to `nftModule.NewAppModule(app.NftKeeper)` but that doesn't seem to be correct.